books/forms.py

```diff
@@ -5,4 +5,14 @@ class PostBookForm(forms.ModelForm):
     
     class Meta:
         model = Book
-        fields = ('title', 'author', 'cover')
\ No newline at end of file
+        fields = ('title', 'author', 'cover')
+
+    def __init__(self, *args, **kwargs):
+        super(PostBookForm, self).__init__(*args, **kwargs)
+        self.fields['cover'].required = False
+
+    def clean_author(self): 
+        data = self.cleaned_data.get('author')
+        if len(data) <= 3:
```

Comment:
Django comes with a `min_length` validator built into (forms.Charfield](https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.CharField).
So you can either set it the way you did above with `required` within `__init__` or you can explicitly define the form field:

```python
class PostBookForm(forms.ModelForm):
    author = forms.Charfield(min_length=3)
    
     # ... everything else remains the same
```